tasks/task456_matres_intention_classification.json

```diff
@@ -11,7 +11,7 @@
     "Domains": [
         "News"
     ],
-    "Definition": "You will be given a context and a verb separated with a \n, and you have to identify if the given verb implies eiher an opinion, an intention, or a wish. The output should be: 'Yes' if the verb irefers to an opinion, an intention, or a wish; and 'No' otherwise. Please note that a verb refers to an intention only if it refers to an event planned to happen in the future and is not conditional (or part of a condition).",
+    "Definition": "You will be given a context and a verb separated with a newline character, and you have to identify if the given verb implies eiher an opinion, an intention, or a wish. Please note that a verb refers to an intention only if it refers to an event planned to happen in the future and is not conditional (or part of a condition). The output should be: 'Yes' if the verb irefers to an opinion, an intention, or a wish; and 'No' otherwise. ",
```

Comment:
The definition can be worded better (the way it is now, can be vague and interpreted as if we have multiple classes: intention, opinion, wish, ...)
Something like this:
`You will be given a context and a verb separated with a newline character. You have to identify verbs that imply an opinion, an intention, or a wish. Please note that a verb refers to an intention only if it refers to an event planned to happen in the future and is not conditional (or part of a condition). The output should be: 'Yes' if the verb refers to an opinion, an intention, or a wish; and 'No' otherwise.`